src/gpgpu_gles.c

```diff
@@ -320,8 +320,98 @@ int GPGPU_API gpgpu_firConvolution2D(float* data, float* kernel, int size, float
 
 int GPGPU_API gpgpu_matrixMultiplication(int* a, int* b, int size, int* res)
```

Comment:
overall I think we could refactor it to not copy over so much code whenever we add an operation. Would you be interested in performing such a refactor?

Reply:
Yes, I would be interested surely. Thank You.

Reply:
Great :)